src/NServiceBus.Core/Pipeline/Incoming/SatelliteTransportReceiveToPhysicalMessageProcessingConnector.cs

```diff
@@ -0,0 +1,15 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Pipeline;
+
+    class SatelliteTransportReceiveToPhysicalMessageProcessingConnector : StageConnector<ITransportReceiveContext, IIncomingPhysicalMessageContext>
```

Comment:
@andreasohlund good proposal!

On Sunday, February 28, 2016, Andreas Öhlund <notifications@github.com>
wrote:

> In
> src/NServiceBus.Core/Pipeline/Incoming/SatelliteTransportReceiveToPhysicalMessageProcessingConnector.cs
> <https://github.com/Particular/NServiceBus/pull/3516#discussion_r54348944>
> :
>
> > @@ -0,0 +1,15 @@
> > +namespace NServiceBus
> > +{
> > +    using System;
> > +    using System.Threading.Tasks;
> > +    using Pipeline;
> > +
> > +    class SatelliteTransportReceiveToPhysicalMessageProcessingConnector : StageConnector<ITransportReceiveContext, IIncomingPhysicalMessageContext>
>
> Since we want the satellite pipeline completely separate from the "main"
> pipeline should we just give it a raw "SatelliteReceiveContext" instead to
> avoid sharing anything with the main pipeline?
>
> This would also remove "physical" from the lingo of the satellite pipeline
> since there isn't any "logicalmessage" part to that pipeline?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3516/files#r54348944>.
>


-- 
*Don't miss the Async/Await Webinar Series*
*• Learn more <http://go.particular.net/l/27442/2016-02-15/681bjl> →*